sonic_platform_base/module_base.py

```diff
@@ -161,6 +174,10 @@ def set_admin_state(self, up):
         The up state will take the module to MODULE_STATUS_PRESENT,
         MODULE_STATUS_FAULT or MODULE_STAUS_ONLINE states.
 
```

Comment:
up state request on MODULE_STATUS_EMPTY would have no effect (i.e. NO-OP).
up state can't take the module to  MODULE_STATUS_PRESENT. 
Module has to be PRESENT for 'up' state to take it to MODULE_STAUS_ONLINE or MODULE_STATUS_FAULT.

In that case, suggest removing MODULE_STATUS_PRESENT from the comment

Reply:
Addressed. Thanks.